### DISCUSSION TITLE
How to make the terminal show only the current working directory and not all the path, (like shown in the screenshot)

### DISCUSSION THREAD

**Discussion by adamzzeeee** [Cyfrin/security-and-auditing-full-course-s23#247] [Q&A]
### **How to change the terminal from this** 

![Screenshot 2024-11-07 at 13-13-06 Phase 1 Scoping Cyfrin Updraft](https://github.com/user-attachments/assets/1ef2e9b8-3201-400d-a4dc-ed4a56ed28e5)

### **To this**

![Screenshot 2024-11-07 at 13-13-21 Phase 1 Scoping Cyfrin Updraft](https://github.com/user-attachments/assets/3915abf4-064d-451f-a19e-f6b28916eeed)

**Comment by EngrPips** [comment]
Hello @adamzzeeee, Read this https://unix.stackexchange.com/questions/381113/how-do-i-shorten-the-current-directory-path-shown-on-terminal#:~:text=The%20%24PWD%20variable%20contains%20the,the%20last%20%2F%20in%20that%20path. Stack Exchange Post and see if it helps you.
